modules/network-observability-histogram-trafficflow.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-histogram-trafficflow_{context}"]
+== Using the histogram
+You can click *Show histogram* to display a toolbar view for visualizing the history of flows as a bar chart. The histogram shows the number of logs over time. You can select a part of the histogram to filter the flows data in the table that follows the toolbar.
```

Comment:
This doesn't sound right, or maybe it does. I don't know. 

the flow data
the flow's data
the data

Reply:
Hmmmmm I think maybe "...filter the network flow data"